### PR TITLE
feat(tab): Allow to render Tab as an anchor node

### PR DESCRIPTION
**Pull request by barteksekula**

### CODE REVIEW

**Comment by barteksekula** [comment]
I signed it!

**Comment by barteksekula** [comment]
not sure why the build failed for me, the failed tests does not seem to be related with my change, but probably they are, I will try to find out why

File: packages/tab/index.tsx [hunk 1]
```diff
@@ -203,7 +206,9 @@ export default class Tab extends React.Component<TabProps, TabState> {
     const {tabIndex, ['aria-selected']: ariaSelected} = this.state;
 
     return (
-      <button
+      // https://github.com/Microsoft/TypeScript/issues/28892
+      // @ts-ignore
+      <Tag
```
Comment (moog16) [inline packages/tab/index.tsx]: can you also add a unit test to ensure that this renders as an anchor with an href?

**Comment by moog16** [comment]
@barteksekula I tried running the screenshot tests (npm start). There are a lot of TS errors resulting in the screenshot tests failing. Can you please take a look?

**Comment by barteksekula** [comment]
@moog16 it's strange because I'm not getting any ts errors after running npm start. 
however travis is saying that my changes caused test failures in for example select/native select.
I'm trying to set up screenshot tests locally so I can test this out myself (I know I should have done that before submitting the PR - sorry for that.. :) but I naively assumed my changes will not cause any of those).

**Comment by moog16** [comment]
@barteksekula this shouldn't be affecting the other screenshot tests. I don't think it should even affect the tab component. I'll look into this.

**Comment by moog16** [comment]
Can you please make sure you are up to date on the latest rc0.14.0 branch?

**Comment by barteksekula** [comment]
@moog16 great, I rebased and now it's all green. thanks! :)

**Comment by moog16** [comment]
Great! Thanks :)

**Comment by barteksekula** [comment]
@moog16 do I have to do anything else to finalize this? closes #920

**Comment by moog16** [comment]
Thanks for your contribution!

**Pull request merged**